user-interface/src/main/java/life/qbic/datamanager/views/navigation/OpenSideNavItem.java

```diff
@@ -0,0 +1,61 @@
+package life.qbic.datamanager.views.navigation;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.shared.HasSuffix;
+import com.vaadin.flow.component.sidenav.SideNavItem;
+import com.vaadin.flow.dom.Element;
+
+public class OpenSideNavItem extends Span implements HasSuffix {
+  private Element labelElement;
```

Comment:
Some Javadocs would be nice here so we can reuse the component if necessary. 
(BTW what could be a solution if the clickable list extension is required is to use the [accordion](https://vaadin.com/docs/latest/components/accordion) component within this component)